<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEPdeleteDialogClass</class>
 <widget class="QDialog" name="CEPdeleteDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CEPdeleteDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditDelete"/>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Delete!</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidgetDelete">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>VIC trees to delete from SAS:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Commands to execute on CEP nodes:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditVICtrees"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableWidgetDelete</tabstop>
  <tabstop>textEditDelete</tabstop>
  <tabstop>lineEditVICtrees</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
